CardsOfOpprotunity/CardsOfOpprotunity/Views/BoardGame/GameViewModel.swift

```diff
@@ -19,6 +19,8 @@ class GameViewModel: ObservableObject {
     @Published var alertMessage: String = ""
     @Published var gameOver: Bool = false
     @Published var showAlert: Bool = false
+    @Published var playerOneRank: Int = 0
```

Comment:
Suggestion, rename the player's score variables to `playerOneScore` instead of `playerOneRank`

Reply:
we already have a playerOneScore which keeps track of total wins, the rank is just the most recent pulled card